examples/02-Maxwell/Maxwell3D_Choke.py

```diff
@@ -191,8 +193,6 @@
 # ----------------
 
 m3d.save_project(os.path.join(temp_folder, "My_Maxwell3d_Choke.aedt"))
-m3d.modeler.fit_all()
-m3d.plot(os.path.join(m3d.working_directory, "Image.jpg"))
```

Comment:
I would leave the plot to make the demo nicer

Reply:
I delete them, because it returns an error, I have it because it was an error from a copy/paste, my example it is only in pre-processing. What they must do?